.github/workflows/test-deploy.yml

```diff
@@ -89,10 +89,24 @@ jobs:
           registry-url: https://registry.npmjs.org
       - run: npm install
       - run:
-          npm ${{ github.event_name == 'release' && 'publish' || 'publish
+          npm ${{ github.event_name == 'push' && 'publish' || 'publish
           --dry-run' }}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  release:
+    if: github.event_name == 'push'
+    needs: packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - id: meta
+        name: Metadata
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/tags/}
+      - uses: softprops/action-gh-release@v1
+        with:
+          name: CML ${{ steps.meta.outputs.tag }}
+          draft: true
```

Comment:
can easily add `files: build/cml-linux` etc from #842 here

Reply:
how @casperdcl ? with comments?

Reply:
I mean after #847 is merged, we can update #842 to use:

```yml
      - uses: softprops/action-gh-release@v1
        with:
          ...
          files: |
            build/cml-linux
            build/cml-macos
```

Reply:
(updated PR to stop @iterative-olivaw reacting to everything :D)